QuantumLink/Common/Networking/Protocol/Requests/AccountRequest.cs

```diff
@@ -30,8 +28,8 @@ public enum AccountOperation
             DeleteAccount
         }
 
-        public readonly AccountOperation Operation;

```

Comment:
Again, I'm not sure why readonly cannot be used. See this stack overflow answer: https://stackoverflow.com/questions/37496738/c-sharp-readonly-vs-get. By using readonly, we define these properties as feilds intead, which are much faster than going around a get and set.

Reply:
Properties are better